### PR TITLE
rebuild or update for missing platforms

### PR DESCRIPTION
**Pull request by boldorider4**
* updating to have dependents of `torchvision` build for python 3.11.
* this is instrumental for building [sentences-transformers:PR5](https://github.com/AnacondaRecipes/sentence-transformers-feedstock/pull/5)

### CODE REVIEW

File: recipe/meta.yaml
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - {{ pin_compatible('numpy') }}
```

File: recipe/meta.yaml
Comment (skupr-anaconda) [inline recipe/meta.yaml:52]: Add please   ` - numpy {{ numpy }}`

File: recipe/meta.yaml
Comment (skupr-anaconda) [inline recipe/meta.yaml:64]: Without `python` in `run` you can't build a python package

**Review by varlackc — COMMENTED** [review]
The issue in `win-64` will have to be resolved before merging or we might have to skip `win-64`

**Review by varlackc — COMMENTED** [review]
There is currently a PR that needs to be removed

  `Rebuild with PyTorch 1.12.1`
  https://github.com/AnacondaRecipes/torchvision-feedstock/pull/8

File: recipe/meta.yaml
Comment (cbouss) [inline recipe/meta.yaml:67]:
```suggestion
```

Is setuptools really required at runtime?

File: recipe/meta.yaml
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - jpeg {{ jpeg }}  # [not win]
```
It's in conda_build_config.yaml

File: recipe/meta.yaml
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - libpng {{ libpng }}
```
It's in conda_build_config.yaml

File: recipe/meta.yaml
Comment (boldorider4) [reply recipe/meta.yaml:67]: I get a weird warning (or error, can't remember off the top of my head). I was also puzzled by this. Adding it as a run dependency seems to have solved the issue.

File: recipe/meta.yaml
Comment (boldorider4) [reply recipe/meta.yaml]: great, thanks!

File: recipe/meta.yaml
Comment (boldorider4) [reply recipe/meta.yaml]: thanks!

File: recipe/meta.yaml
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
Redundant

**Review by cbouss — APPROVED** [review]
LGTM, provided the current run is successful

File: recipe/meta.yaml
Comment (rafaelmartins) [inline recipe/meta.yaml]: `-vvv` may be too much

Comment (boldorider4) [reply recipe/meta.yaml]: Will fix

**Review by varlackc — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**
